src/aggregator/client/config.go

```diff
@@ -207,6 +207,37 @@ func (c *Configuration) NewClientOptions(
 	return opts, nil
 }
 
+// TLSConfiguration contains the TLS configuration
+type TLSConfiguration struct {
+	TLSEnabled         bool    `yaml:"tlsEnabled"`
+	InsecureSkipVerify *bool   `yaml:"insecureSkipVerify"`
```

Comment:
nit: do you really need to use pointers here? I thought it was somewhere in Uber's style guidelines to avoid using pointers in config structures, but I can't really find any references to it.
Keep it or leave it, my personal preference would be to use values instead of pointers in the config structures unless you absolutely have to use a pointer

Reply:
You are right, pointers are not needed here. Sometimes, it can be used for nullable values.